modules/test/playwright/tests/users-admin-web/accountSettingsRoles.spec.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {accountSettingsPageTest} from '../../fixtures/accountSettingsPagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+
+import {expect, mergeTests} from '@playwright/test';
+
+export const test = mergeTests(accountSettingsPageTest, loginTest);
+
+test('LPD-15689 - roles in account settings should have no save button', async ({
+    accountSettingsPage
+}) => {
+    await accountSettingsPage.goToAccountSettingsRoles();
+
+    await expect(
+        accountSettingsPage.saveButton
+    ).toBeHidden({
```

Comment:
This is wrong. We are not hiding the button. The button does not actually exist in the DOM.

Use `.toBeVisible()`

Reply:
https://playwright.dev/docs/test-assertions "tobeHidden" is the test for not visible according to this, do you still want this changed?

Reply:
Yes, if you look at the detailed explanation of "toBeHidden": https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-hidden

```
Ensures that [Locator](https://playwright.dev/docs/api/class-locator) either does not resolve to any DOM node, or resolves to a [non-visible](https://playwright.dev/docs/actionability#visible) one.
```
It will resolve if it exists in the DOM and is just not visible.